src/main/resources/db/migration/V1_4__create_table_oppfolgingstilfelle_arbeidstaker.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE OPPFOLGINGSTILFELLE_ARBEIDSTAKER
+(
+    id                                  SERIAL               PRIMARY KEY,
+    uuid                                VARCHAR(50)          NOT NULL UNIQUE,
+    created_at                          timestamptz          NOT NULL,
+    personident                         VARCHAR(11)          NOT NULL,
+    oppfolgingstilfeller                jsonb                NOT NULL,
```

Comment:
Her lagres listen over Oppfolgingstilfelle for en arbeidstakter med full historikk slik det ser ut i dette øyeblikksbilde.
Type liste med dette:
```
data class OppfolgingstilfelleDTO(
    val start: LocalDate,
    val end: LocalDate,
    val virksomhetsnummerList: List<String>,
)
```